deployment/copilot/traffic-comparator/manifest.yml

```diff
@@ -29,8 +29,8 @@ storage:
       efs:
         id: ${MIGRATION_COMPARATOR_EFS_ID}
 
-cpu: 256       # Number of CPU units for the task.
-memory: 512    # Amount of memory in MiB used by the task.
+cpu: 512       # Number of CPU units for the task.
+memory: 2048    # Amount of memory in MiB used by the task.
```

Comment:
Not for this PR, but it might be worth thinking about how we can thread these values together from a global spot.  Being able to tweak scaling factors in one place could be valuable.